ramls/patron-notice-policy.json

```diff
@@ -207,74 +157,31 @@
             "properties": {
               "sendHow": {
                 "type": "string",
-                "description": "Defines how notice should be sent, before, after or upon",
+                "description": "Defines how notice should be sent, after or upon",
                 "enum": [
                   "Upon At",
-                  "Before",
```

Comment:
Why are we removing this option?

Reply:
Because this option makes no sense in terms of automated fees/fines.

Reply:
So this section is only for automated fees/fines?

Reply:
Yes, at least at the moment.